README.md

```diff
@@ -57,3 +54,6 @@ To publish your changes, you'll need to increment the version number in `./versi
                 * ( done ) print a warning
                 * ( done ) have the build send an email (set build to unstable)
 * ( done ) add a "clean" script that removes tmp, maybe clears out some docker images, too?
+* ( TODO ) does/can/should ci/test work when not run in the image builder container?
```

Comment:
I'm of the opinion that the CI should be as reproducible as possible, so if it works in the image builder, let's continue to do that, unless there is undue overhead with that approach.  However, I can definitely see the utility of running the test in a more portable fashion.  But overall, this bolus of work is really designed to be automated, so if the automation works in the image builder, I'm fine with calling that a job well done.

Reply:
Yeah, at some point I realized that this dependency had cropped up and never made it really nice and neat for local dev again. Just left this in the readme as a breadcrumb in case we do any significant work on this again. Feel free to edit if confusing.